Flow Actions/Assign Role/assignRole.js

```diff
@@ -0,0 +1,20 @@
+(function execute(inputs, outputs) {
+
+    var user = inputs.user_id;
+    var roleSysId = inputs.role_sys_id;
+
+    var userGR = new GlideRecord("sys_user");
+    userGR.addQuery('user_name', user);
```

Comment:
This GlideRecord query could be optimised:
```
var userGR = new GlideRecord("sys_user");
userGR.get('user_name',user);

if(userGR) {
```

Reply:
Or if you decide to use user sys_id as input, it should simply be `userGR.get(user)`.